readme.md

```diff
@@ -1,44 +1,61 @@
 # remark-smartypants
 
+[![v](https://badgen.net/npm/v/remark-smartypants)][npm]
+[![dt](https://badgen.net/npm/dt/remark-smartypants)][npm]
```

Comment:
Not sure if it makes sense to add this text considering that the most important piece of information is missing (which version and how many downloads). Alternatively leave the alt text out completely.

```suggestion
[![package version](https://badgen.net/npm/v/remark-smartypants)][npm]
[![number of downloads](https://badgen.net/npm/dt/remark-smartypants)][npm]
```